components/crud-web-apps/jupyter/manifests/base/deployment_patch.yaml

```diff
@@ -0,0 +1,27 @@
+# TODO(https://github.com/kubeflow/manifests/issues/774): This is a patch
+# that pulls out from core the parts that should be in pulled into stacks.
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: deployment
+spec:
+  template:
+    spec:
+      containers:
+      - name: jupyter-web-app
+        imagePullPolicy: $(policy)
+        env:
```

Comment:
I believe this is the place that the `App_PREFIX` environment variable will need to be placed. https://github.com/kubeflow/kubeflow/pull/5606#issuecomment-787118739
@kimwnasptd